examples/17_video_mobilenet.py

```diff
@@ -57,19 +58,23 @@ def frame_norm(frame, bbox):
         return (np.clip(np.array(bbox), 0, 1) * norm_vals).astype(int)
 
 
-    def to_planar(arr: np.ndarray, shape: tuple) -> list:
-        return [val for channel in cv2.resize(arr, shape).transpose(2, 0, 1) for y_col in channel for val in y_col]
-
+    def to_planar(arr: np.ndarray, shape: tuple) -> np.ndarray:
+        return cv2.resize(arr, shape).transpose(2,0,1).flatten()
 
     cap = cv2.VideoCapture(video_path)
     while cap.isOpened():
         read_correctly, frame = cap.read()
         if not read_correctly:
             break
 
-        nn_data = dai.NNData()
-        nn_data.setLayer("data", to_planar(frame, (300, 300)))
-        q_in.send(nn_data)
+        tstamp = monotonic()
+        data = to_planar(frame, (300, 300))
+        img = dai.ImgFrame()
```

Comment:
ImgFrame constructor sets time to current monotonic time by default. Just a note

Reply:
Sorry to bring bad news again, I'm not sure if it's just because of my host system but I experienced massive performance loss (fps goes down to ~1)  when I changed the code from the original to send `ImgFrame()` data as shown above